transformer_engine/pytorch/module/base.py

```diff
@@ -1070,13 +1073,32 @@ def prepare_forward(
                 if self.training and is_fp8_activation_recompute_enabled():
                     FP8GlobalStateManager.copy_forward_fp8_meta_tensors_for_recompute(self.fp8_meta)
 
-        with get_nvtx_range_context(self.__class__.__name__ + " forward"):
-            if not allow_non_contiguous and not inp.is_contiguous():
-                inp = inp.contiguous()
-            yield inp
+        # with get_nvtx_range_context(self.__class__.__name__ + " forward"):
+        if _nvtx_enabled():
+            torch.cuda.nvtx.range_push(self.__class__.__name__ + " forward")
```

Comment:
`nvtx_range_push`/`nvte_range_pop` does the same thing and is slightly cleaner:
https://github.com/NVIDIA/TransformerEngine/blob/3ff0b8d4934f1459d2ca84a5cb312b9d678b1a5a/transformer_engine/pytorch/utils.py#L614
https://github.com/NVIDIA/TransformerEngine/blob/3ff0b8d4934f1459d2ca84a5cb312b9d678b1a5a/transformer_engine/pytorch/utils.py#L632

```suggestion
        nvtx_range_push(self.__class__.__name__ + " forward")
```

Reply:
Thanks, will change it :-).